sky/shell/platform/mojo/BUILD.gn

```diff
@@ -44,4 +49,11 @@ mojo_native_application("mojo") {
     "//third_party/icu",
     "//url",
   ]
+
+  if (is_android) {
+    ldflags = [
+      "-lGLESv2",
```

Comment:
Actually you don't need a different build - you could split this file out of the //skia target into its own source set and depend on that only in the SkyShell.apk targets and not the sky_shell.mojo targets.  The new target could also put the correct ldflags in whenever it is depended on as a bonus.